src/rendering/.env

```diff
@@ -72,3 +72,7 @@ PERSONALIZE_MIDDLEWARE_CDP_TIMEOUT=
 
 # Timeout (ms) for Sitecore Experience Edge requests to respond within. Default is 250.
 PERSONALIZE_MIDDLEWARE_EDGE_TIMEOUT=
+
+# Search API
+NEXT_PUBLIC_SEARCH_API_DOMAIN=

```

Comment:
What values should we use here? Are they found in the Discover Developer Resources?

Reply:
`SEARCH_API_DOMAIN` should be set to your Sitecore Search "Domain ID".
`SEARCH_API_HOST` should be set to your "SERVER API HOST" in "Developer Resources > API Access > API Hosts" without the URL scheme. E.g.: `api.rfksrv.com`.